engine/userguide/eng-image/baseimages.md

```diff
@@ -67,14 +67,14 @@ NOTE: Because Docker for Mac and Docker for Windows use a Linux VM, you must com
 Then you can run it (on Linux, Mac, or Windows) using: `docker run --rm hello`
 
 This example creates the hello-world image used in the tutorials.
-If you want to test it out, you can clone [the image repo](https://github.com/docker-library/hello-world)
+If you want to test it out, you can clone [the image repo](https://github.com/docker-library/hello-world).
 
 
 ## More resources
 
-There are lots more resources available to help you write your 'Dockerfile`.
+There are lots more resources available to help you write your `Dockerfile`.
 
 * There's a [complete guide to all the instructions](../../reference/builder.md) available for use in a `Dockerfile` in the reference section.
 * To help you write a clear, readable, maintainable `Dockerfile`, we've also
-written a [`Dockerfile` Best Practices guide](dockerfile_best-practices.md).
+written a [`Dockerfile` Best Practices Guide](dockerfile_best-practices.md).
```

Comment:
yes. Our capitalization rules are simple:
* Is the first word in the sentence? Capitalize
* Is it a product name? Capitalize
* Everything else, keep it lower case.